src/appleseed/renderer/modeling/edf/coneedf.cpp

```diff
@@ -195,6 +196,24 @@ namespace
             return sample_cone_uniform_pdf(m_cos_half_angle);
         }
 
+		virtual float get_uncached_max_radiance_value() const
+		{
+			Spectrum radiance;
+
+			Source *source = m_inputs.source("radiance");
+
+			assert(source);
+
+			source->evaluate_uniform(radiance);
```

Comment:
If the source is not uniform (constant), `evaluate_uniform()` will not evaluate it. This will happen for instance is the source is a texture, or a procedural function.

In that case, I think we should return `numeric_limits<float>::max()` to make sure we don't consider this light source a low contributing one.